Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/SettingsAdapter.java

```diff
@@ -286,7 +286,26 @@ public void onSliderClick(SliderSetting item, int position)
     slider.setValueFrom(item.getMin());
     slider.setValueTo(item.getMax());
     slider.setValue(mSeekbarProgress);
-    slider.setStepSize(1);
+
+    // Sliders can get frustrating to use with a small step size and large ranges
+    int maxRange = item.getMax();
+    if (maxRange > 10000)
+    {
+      slider.setStepSize(1000);
+    }
+    else if (maxRange > 1000)
+    {
+      slider.setStepSize(100);
+    }
+    else if (maxRange > 100)
+    {
+      slider.setStepSize(10);
+    }
+    else
+    {
+      slider.setStepSize(1);
+    }
+
```

Comment:
Can be generalized like this (untested):

```suggestion
    if (maxRange <= 100)
    {
      slider.setStepSize(1);
    }
    else
    {
      slider.setStepSize(Math.pow(Math.ceil(Math.log(maxRange)) - 2, 10));
    }
```